examples/SampleApp/src/screens/AdvancedUserSelectorScreen.tsx

```diff
@@ -102,13 +114,18 @@ export const LabeledTextInput: React.FC<LabeledTextInputProps> = ({
         </Text>
       )}
       <TextInput
+        onBlur={onBlur}
         onChangeText={onChangeText}
+        onFocus={onFocus}
         placeholder={label}
         placeholderTextColor={grey}
         returnKeyType='next'
         style={[
           styles.input,
-          { color: black, fontWeight: value ? undefined : '500' }, // design team wanted placeholder fontWeight of 500
+          {
+            color: black,
+            fontWeight: value ? undefined : '500',
+          }, // design team wanted placeholder fontWeight of 500
```

Comment:
```suggestion
            color: black,
            fontWeight: isEmpty && '500',
          }, // design team wanted placeholder fontWeight of 500
```

Reply:
This gives an error:

` Types of property 'fontWeight' are incompatible.`

Reply:
`isEmpty ? 500 : 'normal'` should work